lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -1590,16 +1604,33 @@ instance PersistState (Rnd.RndState t) where
             , gen = gen
             }
 
+    deleteState (wid, sl) = deleteWhere
```

Comment:
At first, I thought that we can reuse this delete with sequential addresses, but maybe it is not worth pursuing.